### PR TITLE
add RMV profile

### PR DESCRIPTION
**Pull request by derhuerst**

### CODE REVIEW

File: p/rmv/example.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+'use strict'
+
+const createClient = require('../..')
+const hvvProfile = require('.')
+
+const client = createClient(hvvProfile, 'hafas-client-example')
```
Comment (juliuste) [inline p/rmv/example.js]:
```suggestion
const rmvProfile = require('.')

const client = createClient(rmvProfile, 'hafas-client-example')
```

File: test/e2e/rmv.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+	validate(t, trip, 'trip', 'trip')
+	t.end()
+})
+
+test('arrivals at Wiesbaden Hbf', async (t) => {
+	const arrivals = await client.arrivals(wiesbadenHbf, {
+		duration: 10, when
+	})
+
+	validate(t, arrivals, 'arrivals', 'arrivals')
```
Comment (juliuste) [inline test/e2e/rmv.js]: Is there any reason you don't use the `testArrivals` method that's imported above here?

Comment (derhuerst) [reply test/e2e/rmv.js]: `testArrivals` asserts that the resulting arrivals have the `stop.id`/`stop.station.id` that we passed in as `id`. Some endpoints, such as the RMV one, use varying stop IDs though.

Comment (derhuerst) [reply test/e2e/rmv.js]: The test tooling is bad right now, will eventually™ improve it.

**Pull request merged**